agent/org.linkedin.glu.agent-server-upgrade/src/cmdline/resources/bin/agentctl.sh

```diff
@@ -202,14 +204,19 @@ get_pid()
     PID=`cat $PID_FILE`
     PS_STAT=`ps -p $PID -o'user,pid=' | tail -1 | awk '{print $2}'`
 
-    case "$PS_STAT" in
-      $PID  ) echo $PID
-              ;;
-      *     ) echo 0
-              ;;
-    esac
+    if [[ "$PS_STAT" -eq "$PID" ]]; then
+      CMD=$(ps -p $PID -o 'comm=')
```

Comment:
Thank you for the explanation. 

* Is `basename` a standard command on Mac/Linux/Unix?
* not sure if that would help but one of the command line argument is this one: `-Dorg.linkedin.app.name=org.linkedin.glu.agent-server` so if there is an easier way to check for this vs simply checking for `java` then you would be more certain that it is the agent...